packages/node_modules/@ciscospark/react-container-activity-list/src/selectors.js

```diff
@@ -107,15 +107,20 @@ export const getActivityList = createSelector(
     const activityFailures = sendingActivities.get('activityFailures');
     const visibleActivityList = [];
     const now = moment();
-    let lastActorId, lastDay, lastVerb;
+    let lastActorId, lastDay, lastMinute, lastVerb;
     let shouldDisplayNewMessageMarker = false;
 
     conversationActivities.forEach((activity) => {
       if (isActivityVisible(activity)) {
         // Insert day separator if this activity and last one happen on a different day
-        const activityMoment = moment(activity.published, moment.ISO_8601);
-        const activityDay = activityMoment.endOf('day');
+        const activityMomentDay = moment(activity.published, moment.ISO_8601);
```

Comment:
Could you just do:
```js
const activityDay = moment(activity.published, moment.ISO_8601).endOf('day');
```
that way we won't have the confusion of moment changing its base object?

Reply:
Good point, I'll change that